yt_dlp/YoutubeDL.py

```diff
@@ -886,6 +886,7 @@ def trouble(self, message=None, tb=None, is_error=True):
         OPTION='bold',
         EMPHASIS='light blue',
         REQUIREMENT='light blue',
+        FILENAME='green',
```

Comment:
I understand, but going forward, this is better to have here as well.

Also I'm not sure how much work left, I'd like to have it merge ready, in case I decide I can't do it all at once.